.gitlab-ci.yml

```diff
@@ -0,0 +1,81 @@
+stages:
+  - prep
+  - build
+  - test
+  - push
+
+variables:
+  GIT_STRATEGY: fetch
+  COMPOSE_PROJECT_NAME: "kitsune-${CI_JOB_ID}-${CI_COMMIT_SHORT_SHA}"
+
+lint l10n:
+  only:
+    - master
+  tags:
+    - meao
+  stage: prep
+  script:
+    - scripts/lint-l10n-repo.sh --push
+
+docker build:
+  tags:
+    - meao
+  stage: build
+  script:
+    - make build-ci
+
+docker push:
+  tags:
+    - meao
+  stage: test
```

Comment:
Is it should be test? should we push always without test get passed?

Reply:
that's what we're doing now. the idea is that if the tests don't pass we can pull down the image and more easily figure out what's wrong. We only push the `:latest` tags after tests pass.